### PR TITLE
[PVR][guiinfo][Estuary] New GUI info bool: PVR.IsPlayingActiveRecording

### PR DESCRIPTION
**Pull request by jahutchi**
## Description
This adds a new bool: PVR.IsPlayingActiveRecording which will be set to true when kodi is currently playing a recording that is In-Progress.

## Motivation and Context
Currently, there are bools that can be used in the skin to tell whether a recording is playing: Pvr.IsPlayingRecording, or if there is currently a recording in progress (which could be on some other channel): Pvr.IsRecording.

There is also Pvr.IsRecordingPlayingChannel, but this is only set to true when you play a channel (livetv) which has an in-progress recording; and not when you playback an in-progress recording from the start (i.e. from the PVRRecordings window)

## How Has This Been Tested?
Playback an In-Progress recording from the PVR Recordings window & press the "Info" button.

Prior to this patch, there was no indication that you are playing an in-progress recording.

After this patch, you will see that the "Currently Recording" text and icon is displayed on the bottom bar, so you can now tell that you are playing back a recording that is in-progress.

## Screenshots (if appropriate):

Prior to the patch:
screenshot016

After the patch
screenshot017

Note: Ignore the "Noise" top-right of the screen - not sure why this occurred in the screenshot as it didn't display this way on-screen. Take notice of the icon bottom-left, which now shows for this in-progress recording.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [x] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by ksooo** [comment]
Very nice.

Just a remark: You can already see that you are playing an active recording in the info display, because the total time displayed (the yy:yy in "xx:xx / yy:yy" display at the bottom) will continously be increasing as long as the recording is active.

Ofc, the new red dot is far more obvious. Definitely an improvement wrt. convenience.

File: addons/skin.estuary/xml/DialogSeekBar.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
 					</include>
 				</control>
 				<control type="group">
-					<visible>PVR.IsRecordingPlayingChannel + !Player.ChannelPreviewActive</visible>
+					<visible>[PVR.IsRecordingPlayingChannel | PVR.IsPlayingActiveRecording] + !Player.ChannelPreviewActive</visible>
```
[the remark this answers is not in the document]
Comment (jahutchi) [reply addons/skin.estuary/xml/DialogSeekBar.xml]: Good spot

**Comment by jahutchi** [comment]
Suggested changes have been made

**Review by ksooo — APPROVED** [review]
Thank you for your contribution.

**Comment by ksooo** [comment]
@ronie could you please announce the new guiinfo bool in the forum? Thanks.

**Comment by gade01** [comment]
> @ronie could you please announce the new guiinfo bool in the forum? Thanks.

I just announced the bool in the forum and updated the wiki.

**Comment by ksooo** [comment]
@gade01 thank you. 👍

**Pull request merged**